apps/meteor/app/models/server/models/LivechatRooms.js

```diff
@@ -9,6 +9,7 @@ export class LivechatRooms extends Base {
 	constructor(...args) {
 		super(...args);
 
+		this.tryEnsureIndex({ _id: 1, t: 1 });
```

Comment:
if the query is already using `_id: 'something'` which is also unique, this might not be needed, right?

Reply:
Ouch, this shouldn't be here, thanks!